client/my-app/src/app/workflow/page.tsx

```diff
@@ -0,0 +1,75 @@
+"use client"
+import { useState } from 'react'
+import { Input } from "@/components/ui/input"
+import { Check } from 'lucide-react'
+import { Button } from "@/components/ui/button"
+import { Card } from "@/components/ui/card"
+import { Badge } from "@/components/ui/badge"
+
+interface WorkflowStepsProps {
+    prName: string;
+}
+
+export default function WorkflowSteps({prName = "PullMate"}: WorkflowStepsProps) {
+    const [showEmailInput, setShowEmailInput] = useState(false)
```

Comment:
since this is a page not a single component, 
can we do this as .. taking the repo name from the url params instead of taking in props ??

Reply:
done